tests/renderer/components/commands-publish-button-spec.tsx

```diff
@@ -124,6 +124,7 @@ describe('Action button component', () => {
     beforeEach(() => {
       // create a button that's primed to publish a new gist
       ({ instance } = createActionButton());
+      state.isPublishingGistAsRevision = false;
```

Comment:
I opted not to add a test for when `isPublishingGistAsRevision = true` since we mock out octokit- let me know if it's worth it to find a way to see if updateGist() is called with the right params when publishGist() is called with `isPublishingGistAsRevision = true`

Reply:
update: made a test 😛